homeassistant/components/zeversolar/sensor.py

```diff
@@ -0,0 +1,100 @@
+"""Support for the JustNimbus platform."""
+from __future__ import annotations
+
+from collections.abc import Callable
+from dataclasses import dataclass
+from typing import Any
+
+from homeassistant.components.sensor import (
+    SensorDeviceClass,
+    SensorEntity,
+    SensorEntityDescription,
+    SensorStateClass,
+)
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import CONF_IP_ADDRESS, UnitOfEnergy, UnitOfPower
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity import EntityCategory
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+from homeassistant.helpers.typing import StateType
+
+from .const import DOMAIN
+from .coordinator import ZeverSolarCoordinator
+from .entity import ZeverSolarEntity
+
+
+@dataclass
+class ZeverSolarEntityDescriptionMixin:
+    """Mixin for required keys."""
+
+    value_fn: Callable[[ZeverSolarCoordinator], Any]
+
+
+@dataclass
+class ZeverSolarEntityDescription(
+    SensorEntityDescription, ZeverSolarEntityDescriptionMixin
+):
+    """Describes ZeverSolar sensor entity."""
+
+
+SENSOR_TYPES = (
+    ZeverSolarEntityDescription(
+        key="pac",
+        name="Power in watts",
```

Comment:
Just Power would be enough, right? As in, the unit of measurement is already part of the entity, we don't have to describe that?

```suggestion
        name="Power",
```

Reply:
Alternatively, assuming this is describing current power

```suggestion
        name="Current power",
```